### PR TITLE
[10.0] [ADD] analytic_schedule

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
Analytic Schedule
=================

The project / analytic account's start and end dates are automatically computed based on the earliest start and latest finish date of the children.

### CODE REVIEW

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Tested as follows:
Created a parent project without dates. Create a child. Add the dates to the child. The dates of the parent remained blank.

**Comment by nikul-serpentcs** [comment]
@aheficent Added a unit test case, Could you please review.

File: analytic_schedule/tests/test_account_analytic_account.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    def setUp(self):
+        super(TestAccountAnalyticAccount, self).setUp()
+        self.analytic_account_obj = self.env['account.analytic.account']
+        self.analytic_line_obj = self.env['account.analytic.line']
+        self.partner1 = self.env.ref('base.res_partner_1')
+        self.partner2 = self.env.ref('base.res_partner_2')
+        self.analytic_parent = self.analytic_account_obj.create({
+            'name': 'Test Parent',
+            'code': '01',
+        })
```
Comment (AaronHForgeFlow) [inline analytic_schedule/tests/test_account_analytic_account.py:19]: Can you create another child with different dates here?
Thanks

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]

**Comment by nikul-serpentcs** [comment]
@aheficent Created another child with different dates, Could you please look it.

**Comment by AaronHForgeFlow** [comment]
As in v10 the dates are hold in the project. This module has to be refactored to work with projects instead. As the module name should change I'll close this PR and I'll do the thing in another PR.

**Comment by AaronHForgeFlow** [comment]
@nikul-serpentcs as we have now in v10 the dates in the project when changing the date of the analytic account we should change also the project that has that analytic_account_id associated. Can you do that?

Also, can you remove the copyright thing?

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please look it.

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Can you also change the analytic dates when the project dates change?

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please look it.

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
It is not working. When changing the project dates, the analytic dates are modified, so it will trigger set back the analytic dates to the project.

**Comment by AaronHForgeFlow** [comment]
This module should be deprecated in v10. It's complex to maintain and does not add any feature. Closed PR. I'm sorry for this.

**Pull request closed without merging**